packages/dbos-cloud/applications/deploy-app-code.ts

```diff
@@ -223,11 +224,14 @@ export async function deployAppCode(
           applicationAvailable = true;
         }
       }
-      await sleepms(1000);
+      await sleepms(100);
     }
-    await sleepms(5000); // Leave time for route cache updates
+    await sleepms(1000); // Leave time for route cache updates
```

Comment:
Nice catch!  Can we eliminate this entirely?

Reply:
In the cloud , if it is not there is the cache, we do read the assigned host from the db. In theory it should work and in my test run it seems to work. But let us see what @maxdml thinks

Reply:
We can remove it if we are OK with requests sent to the app _right after the deploy command completes_ can fail with a 502 bad gateway. This might fail some e2e in the cloud.

Reply:
@maxdml If it is not there in the cache , do we not look up from the db and use it / push it in cache ?